scripts/sharpen_bottom_camera.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import rospy
+import cv2
+import numpy as np
+from cv_bridge import CvBridge, CvBridgeError
+
+from sensor_msgs.msg import Image
+
+from opencv_display_mult import im_show_m
+
+bridge = CvBridge()
+
+def image_callback(data):
+
+    try:
+      image = bridge.imgmsg_to_cv2(data, "rgb8")
```

Comment:
are these indents correct? same as on line 19.

Reply:
This is a POC script it doesn't really matter.